test/FunctionalTests/Microsoft.OData.Core.Tests/ScenarioTests/Roundtrip/ContextUrlWriterReaderTests.cs

```diff
@@ -911,7 +912,7 @@ public void ExpandedEntity()
                     writer.WriteStart(entry);
                     writer.WriteEnd();
                 },
-                string.Format("\"{0}$metadata#Employees(AssociatedCompany,AssociatedCompany())/$entity\"", TestBaseUri),

```

Comment:
>,AssociatedCompany()) [](start = 73, length = 21)

Is it in v401?

Reply:
The default version is v4 for context uri. For this test, since the version is not explicitly passed, the context uri is for v4.  

---
In reply to: [258205555](https://github.com/OData/odata.net/pull/1407#discussion_r258205555) [](ancestors = 258205555)